Makefile

```
@@ -19,10 +19,16 @@ prod:
 	$(eval export db_plan=small-ha-11)
 	@true
 
-.PHONY: require_env_stub build push deploy setup_paas_env
+.PHONY: require_env_stub build push deploy setup_paas_env setup_paas_db setup_paas_app
+
 require_env_stub:
 	test ${env_stub} || (echo ">> env_stub is not set (${env_stub})- please use make dev|staging|prod (task)"; exit 1)
 
+get_git_status:
+	$(eval export git_commit_sha=$(shell git rev-parse HEAD))
```

```suggestion
	$(eval export git_commit_sha=$(shell git rev-parse --short HEAD))
```

the short SHA is a bit easier on the eyes...